hawkbit-sdk/hawkbit-sdk-dmf/src/main/java/org/eclipse/hawkbit/sdk/dmf/UpdateHandler.java

```diff
@@ -97,6 +97,7 @@ public UpdateProcessor(
 
         @Override
         public void run() {
+            log.info("Sending feedback for {}", dmfController.getControllerId());
```

Comment:
dmfController.sendFeedback will also log this

Reply:
removed it.